dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/HostFixture.cs

```diff
@@ -50,6 +51,8 @@ public HostFixture(TestConfiguration testConfiguration, DbHelper dbHelper)
 
     public Mock<IZendeskApiWrapper> ZendeskApiWrapper { get; } = new Mock<IZendeskApiWrapper>();
 
+    public Mock<IUserImportCsvStorageService> UserImportCsvStorageService { get; } = new Mock<IUserImportCsvStorageService>();
```

Comment:
I think a line needs adding to `HostFixture.ResetMocks` for this new mock

Reply:
Done